wiki/Community/Project_Loved/en.md

```diff
@@ -64,20 +57,14 @@ Every month, beatmaps are chosen according to 9 categories, with the goal of ens
 
 Additionally, up to three *wildcard* beatmaps may be chosen from any category, adding to a total of twelve beatmaps per month.
 
-For any questions regarding what goes in each category, contact the osu! Loved captains.
+For any questions regarding what goes in each category, contact the [osu! Loved captains](https://osu.ppy.sh/groups/31?mode=osu).
 
 #### Rules
 
 **All rules are exactly that: rules.** They are not guidelines and may not be broken under any circumstance.
 
-- **The beatmap must abide by the [beatmap submission rules](/wiki/Rules#beatmap-submission-rules).**
-- **The beatmap must not be currently in the Qualified, Ranked or Approved categories.** Ideally, a map should not be going for Ranked and Loved at the same time, as they are mutually exclusive.
 - **The beatmap must be submitted at least 1 year before Loved nomination, except for beatmaps that break ranking criteria due to their concept.** Submitting a beatmap solely for Loved is discouraged when Ranked might be a better option. Exceptions include sliderator usage, Aspire maps, storyboard gameplay gimmicks, etc.
 - **The beatmap must have at least 30 favourites.** This is a minimum threshold to indicate a positive and existing community reaction to the beatmap. Brigading for favourites is discouraged, although captains cannot verify whether that happens.
-- **Every person involved in the beatmap must not disapprove of it being Loved.** This includes guest difficulty creators, hitsounders, skinners, and storyboarders. If any user involved in the beatmap explicitly disagrees with their contribution being Loved, then it is to be removed from the beatmap after its nomination.
-- **The beatmap host cannot be restricted.** Guest difficulties from restricted users will be judged on a case-by-case basis once the map is considered for nomination by a Loved captain.
-- **At least one difficulty of the beatmap relevant to the Loved submission process must not auto-fail.** Leaderboards are pointless for beatmaps with broken score submission. For the same reason, this also forbids beatmaps that are relevant solely because of an unpassable difficulty, even if they have other difficulties. If the beatmap has a difficulty that auto-fails, a new difficulty with the problematic hit objects removed needs to be also included, preferably removing the original.
-- **Every difficulty of a beatmap must have a minimum drain time of 20 seconds.** This is lower than the 30-second minimum of the ranking criteria for the sake of being less limiting.
 - **A beatmap must have at least some form of mapping contribution from the person submitting it.** This specifically avoids "beatmap compilations" which, even with the original mappers' permissions, do not represent the kind of original work showcased in Loved.
```

Comment:
clarify? this would only apply to pending, wip or graveyarded beatmaps in the compilations, since ranked, loved or approved maps are disallowed by principle of the rules already mentioned above.

Reply:
resolved ondiscord